options_compile.cpp

```diff
@@ -161,6 +161,7 @@ std::string EffectiveOptionsFilter::processOptions(const OpenCLArgList &args,
       // The below assert is usable for manual debugging only
       // default:
       // assert(false && "some unknown argument");
+    case OPT_COMPILE_coverage:
```

Comment:
is line-tables-only enough for coverage?

Reply:
Yes, line-tables-only is enough. Code coverage doesn't need other information like type.